### PR TITLE
Move Teads ads between paragraphs 

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?

For desktop articles with h2 headings Teads ads are appearing between h2 and p (check the Before screenshot) making the article hard to read. 
This is an attempt to fix it by changing the spacefinder rules on desktop's inline1 slots.
Sets it to 1% AB test to check all edge cases and measure the impact on ads impressions

### **Before**

Screenshot 2019-09-13 at 16 53 49

### **After**
Screenshot 2019-09-13 at 16 53 31

## What is the value of this and can you measure success?
Will improve readability of articles with headings

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Review by ripecosta — APPROVED** [review]

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -105,4 +105,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2020, 7, 30),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-commercial-inline1-headings",
+    "Test inline1 ads to not be placed right after h2 headings",
+    owners = Seq(Owner.withGithub("ioanna0")),
+    safeState = On,
```
Comment (katebee) [inline common/app/conf/switches/ABTestSwitches.scala]: Does `safeState` have to be `On`? Most of our tests have safeState as `Off` since this restores default behaviour.

Comment (ioanna0) [reply common/app/conf/switches/ABTestSwitches.scala]: Thanks for pointing it out, have changed it to Off
